OracleHTTPServer/dockerfiles/12.2.1.2.0/Dockerfile

```diff
@@ -67,21 +70,39 @@ RUN yum install -y \
         gcc gcc-c++ glibc glibc-devel libaio libaio-devel libgcc \
 	libstdc++ libstdc++-devel ksh make ocfs2-tools sysstat && \
     yum clean all && \
-    rm -rf /var/cache/yum/* && \
     chmod a+xr /u01 && \
     useradd -b /u01 -m -s /bin/bash oracle && \
     echo oracle:oracle | chpasswd && \
-    mkdir /u01/oracle/.inventory
+    mkdir /u01/oracle/.inventory && \
+    chown oracle:oracle -R /u01 
 
+# Install and configure Oracle JDK
 # Go to /u01 as user 'oracle' to proceed with OHS installation
 #--------------------------------------------------------
-RUN unzip -q -d /u01 /u01/$OHS_PKG && \
-    chown oracle:oracle -R /u01 && \
-    su -c "/u01/$OHS_BIN -ignoreSysPrereqs -novalidation -silent -responseFile /u01/install.file -invPtrLoc /u01/oraInst.loc -jreLoc $JAVA_HOME ORACLE_HOME=$ORACLE_HOME" - oracle && \
-    rm /u01/$OHS_PKG /u01/$OHS_BIN /u01/fmw_*_readme.htm /u01/oraInst.loc /u01/install.file
-
 USER oracle
-WORKDIR /u01/oracle/
+RUN cd /u01 && unzip -q /u01/$OHS_PKG && cd - && \
+    /u01/$OHS_BIN -ignoreSysPrereqs -novalidation -silent -responseFile /u01/install.file -invPtrLoc /u01/oraInst.loc ORACLE_HOME=$ORACLE_HOME && \ 
+    rm /u01/$OHS_BIN /u01/$OHS_PKG /u01/oraInst.loc /u01/install.file && \
+    mkdir /u01/oracle/container-scripts/ /u01/oracle/logs 
+
+# Copy required files to build this image
+# ------------------------------------------------------
+COPY container-scripts/* /u01/oracle/container-scripts/
+
+# Configuration of OHS domain
+# Copy static html for sanity check
+#---------------------------------------------------
+RUN echo "export PATH=$PATH:$ORACLE_HOME/oracle_common/common/bin:$ORACLE_HOME/user_projects/domains/ohsDomain/bin" >> /u01/oracle/.bashrc 
```

Comment:
I'm not entire sure what the worth of this is: a Docker container rarely runs `bash` so modifying the `.bashrc` file for the `oracle` user would only trigger if someone manually exec'd `bash` in a running container.